### ISSUE TITLE
deconz discover: Cannot read properties of undefined (reading 'name')

### ISSUE THREAD

**Issue by edwinveger** [ebaauw/homebridge-deconz#117]
Hi, on a new installation of homebridge-deconz I'm getting the following error when running `deconz discover`.

```
pi@homebridge:/var/lib/homebridge $ deconz discover -S
deconz: fatal: unhandled rejection: TypeError: Cannot read properties of undefined (reading 'name')
    at Discovery.<anonymous> (/var/lib/homebridge/node_modules/homebridge-deconz/cli/deconz.js:453:50)
    at Discovery.emit (node:events:513:28)
    at /var/lib/homebridge/node_modules/homebridge-deconz/lib/Deconz/Discovery.js:162:18
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

I have Raspberry Pi 2B with a Conbee ii USB stick.
I've installed the Homebridge image with Raspberry Pi Imager.
On `homebridge.local:4530` I can see the Conbee as `Phoscon-GW`, so deconz itself appears to work (I have also paired a light which I can control from Phoscon).
But the homebridge plugin cannot find the gateway, I'm getting a `warning: 0000000000000000: not a RaspBee/ConBee mac address` log message every 5 minutes. Hence I was trying to discover the gateway myself to update the config JSON, but encountered this issue.

```
pi@homebridge:/var/lib/homebridge $ deconz config
deconz: localhost: request 1: GET /config
deconz: warning: localhost: request 1: http status 404 Not Found
deconz: fatal: localhost: deCONZ gateway not found
```

More information:
 - [deCONZ] homebridge-deconz v0.1.4, node v18.14.1, homebridge v1.6.0, homebridge-lib v6.3.8
 - Homebridge v1.6.0 (HAP v0.11.0) (Homebridge B181) is running on port 51405.
 - hardware: Raspberry Pi 2B 1.1 (1GB)

Is there something I should configure differently, or is this a bug?

**Comment by ebaauw** [comment]
> warning: 0000000000000000: not a RaspBee/ConBee mac address

The deCONZ gateway hasn't initialised properly, and reports all `0`s instead of the ZIgbee mac address of the RaspBee or ConBee.  This is an issue with deCONZ, but I don't know what causes this.

> TypeError: Cannot read properties of undefined (reading 'name')

That's actually a bug in the error handler.  It tries to log `0000000000000000: not a RaspBee/ConBee mac address`, but this error is not the result of a request, so it cannot get the `name` of the server the request was sent to.

**Comment by ebaauw** [comment]
Fixed in v1.0.5.

**Comment by edwinveger** [comment]
Thanks for the quick response and fixing the bug 👍🏻 

I'll continue to investigate why the deCONZ gateway isn't fully up.
